tier4/{{cookiecutter.project_slug}}/.github/codejson/hooks/post_gen_project.py

```diff
@@ -0,0 +1,92 @@
+import subprocess
+import shutil
+import json
+import os
+import shutil
+
+def get_date_fields():
+    # Run git commands and capture as string
+    output = subprocess.run(['git', 'log', '--date=iso', '--pretty=%cI', '--max-parents=0', '-n', '1'], capture_output=True, text=True)
+
+    # Store string and strip of leading / trailing whitespace
+    date =  output.stdout.strip()
+
+    # Create a dictionary for date information to be pushed to JSON
+    date_information = {"created": f"{date}", 
+    "lastModified": "{% now 'utc', '%Y-%m-%dT%H:%M:%S%z' %}",
+    "metadataLastUpdated": "{% now 'utc', '%Y-%m-%dT%H:%M:%S%z' %}"}
+
+    return date_information
+
+def update_code_json(json_file_path):
+    # Read the JSON 
+    with open(json_file_path, 'r') as file:
+        data = json.load(file)
+
+    # Add date_information and labor hours to the JSON
+    data['date'] = get_date_fields()
+    data['laborHours'] = get_scc_labor_hours()
+
+    # Update the JSON 
+    with open(json_file_path, 'w') as file:
+        json.dump(data, file, indent = 2)
+
+def get_scc_labor_hours():
+    if shutil.which('scc') is not None:
+        try:
+            #Run scc and load results into a dictionary
+            #assuming we are in the .git directory of the repo
+            d = json.loads(subprocess.run(["scc","..", "--format","json2"],check=True, capture_output=True).stdout)
+            
+
+            return d['estimatedScheduleMonths'] * 730.001
```

Comment:
Could we round the number to a whole number or maybe 1 or 2 decimal points? The code.json schema says `laborHours` should be a number but they didn't specify what kind of number...
https://github.com/GSA/code-gov-data/blob/master/schemas/schema-2.0.0.json#L188
<img width="337" alt="Screenshot 2024-10-01 at 2 44 21 PM" src="https://github.com/user-attachments/assets/06c896d5-62ce-43a1-aa1b-2cc595ff6447">